### DISCUSSION TITLE
Source() .qmd files

### DISCUSSION THREAD

**Discussion by nipnipj** [quarto-dev/quarto-cli#2541] [Q&A] [answered]
Is there a way to `source()` small `.qmd` files into a main `.qmd` file?

**Comment by mcanouil** [comment, marked as answer]
Yes, the include shortcode: `{{< include small.qmd >}}`.
Note that, the included Quarto document should not have yaml header, unless you want to override the yaml header of the main document.

https://quarto.org/docs/authoring/includes.html
